Revit_Core_Engine/Convert/Revit/FromRevit/Parameter.cs

```diff
@@ -32,11 +32,19 @@ public static partial class Convert
         /****               Public Methods              ****/
         /***************************************************/
 
-        public static RevitParameter ParameterFromRevit(this Parameter parameter, IEnumerable<IParameterLink> parameterLinks = null)
+        public static RevitParameter ParameterFromRevit(this Parameter parameter, IEnumerable<IParameterLink> parameterLinks = null, bool onlyLinked = false)
         {
             if (parameter == null)
                 return null;
 
+            string name = parameter.Definition.Name;
+
+            IParameterLink parameterLink = parameterLinks.ParameterLink(parameter);
```

Comment:
Can I check, is this anything to do with parameter links? 🤣

Reply:
Em, I think it is more about parameter links. 🐒

Reply:
I guess that makes some sense 🤔